keps/0000-kep-template.md

```diff
@@ -86,9 +83,24 @@ A table of contents is helpful for quickly jumping to sections of a KEP and for
 
 [Tools for generating]: https://github.com/ekalinin/github-markdown-toc
 
+**ACTION REQUIRED:** There must be an issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) referencing this KEP and targeting a release milestone **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)
+of the targeted release**.
+
+## Release Signoff Checklist
+
+Check these off as they are completed for the Release Team to track. These checklist items _must_ be updated for the enhancement to be released.
```

Comment:
Does this checklist imply a different KEP is needed for each stage of maturity: alpha, beta, GA?

Reply:
I don't think we want a different KEP for each of alpha, beta, and GA. The context is lost along the way. How can the release checklist reflect multiple releases?

Reply:
Added a note that this checklist should be reviewed and updated when targeting a new milestone.